doc/api/fs.md

```diff
@@ -3863,6 +3863,52 @@ changes:
 For detailed information, see the documentation of the asynchronous version of
 this API: [`fs.write(fd, string...)`][].
 
+## fs.writev(fd, buffers[, position], callback)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fd` {integer}
+* `buffers` {ArrayBufferView[]}
+* `position` {integer}
+* `callback` {Function}
+  * `err` {Error}
+  * `bytesWritten` {integer}
+  * `buffers` {ArrayBufferView[]}
+
+Write an array of `ArrayBufferView` to the file specified by `fd` using `writev`.
+
+`position` refers to the offset from the beginning of the file where this data
+should be written. If `typeof position !== 'number'`, the data will be written
+at the current position.
+
+The callback will be given three arguments `(err, bytesWritten, buffers)` where
+`bytesWritten` specifies how many _bytes_ were written from `buffers`.
```

Comment:
```suggestion
`bytesWritten` is how many bytes were written from `buffers`.
```

(Like all the other suggestions I'm making, this does not need to block landing this.)